src/App.js

```diff
@@ -170,31 +171,38 @@ const returnLoadedWalletComponent = (Component, props) => {
     return <Redirect to={{ pathname: '/locked/' }} />;
   }
 
+  if (isServerScreen) {
+    // We allow server screen to be shown from locked screen to allow the user to
+    // change the server before from a locked wallet.
+    return returnDefaultComponent(Component, props);
+  }
+
   const reduxState = store.getState();
 
   // Check version
-  if (reduxState.isVersionAllowed === undefined && !isServerScreen) {
+  if (reduxState.isVersionAllowed === undefined) {
     // We already handle all js errors in general and open an error modal to the user
     // so there is no need to catch the promise error below
-    version.checkApiVersion(reduxState.wallet);
+    version.checkApiVersion();
```

Comment:
removed the need for a wallet, since we need to call it from the locked screen as well